Build.fsx

```diff
@@ -183,9 +184,7 @@ Target "CleanNuGetPackages" (fun _ ->
 )
 
 Target "NuGetPack" (fun _ ->
-    let version = "Src/AutoFixture/bin/Release/Ploeh.AutoFixture.dll"
-                  |> GetAssemblyVersion
-                  |> (fun v -> sprintf "%i.%i.%i" v.Major v.Minor v.Build)
+    let version = FileVersionInfo.GetVersionInfo("Src/AutoFixture/bin/Release/Ploeh.AutoFixture.dll").ProductVersion
```

Comment:
My initial thoughts were to decouple build from packaging, so that for the NuGet pack task knows nothing about how version is resolved - we simply take version from the DLL.

However, given that NuGet pack target has dependency on the build, it could be never triggered separately, it might make sense to use global variable instead.